collab-document/src/document.rs

```diff
@@ -14,11 +18,10 @@ const CHILDREN_MAP: &str = "children_map";
 pub struct Document {
   #[allow(dead_code)]
   inner: Collab,
-  root: MapRefWrapper,
-  text_map: TextMap,
-  children_map: ChildrenMap,
+  pub root: MapRefWrapper,
+  pub text_map: TextMap,
+  pub children_map: ChildrenMap,
   pub blocks: BlockMap,
```

Comment:
@appflowy I want to use  `trait` to define the blocks, like

```rs
pub trait OperableBlocks {}
// ...
pub blocks: OperableBlocks
```

However, I encountered a compiler error when trying to use Box<dyn OperableBlocks>. Do you have any suggestions for how to fix it?